extra-radio/aprx/autobuild/defines

```diff
@@ -0,0 +1,6 @@
+PKGNAME=aprx
+PKGSEC=hamradio
+PKGDES="A highly configurable APRS I-gate/Digipeater Daemon"
```

Comment:
Try not to advertise the upstream's claims, they might say it's "highly configurable," but not everyone will agree.